packages/eth2.0-state-transition/src/index.ts

```diff
@@ -24,14 +24,15 @@ export function stateTransition(
   state: BeaconState,
```

Comment:
Can you add a block comment above saying what each option does/is for?
Will be helpful for future us and for autogenerating docs.